tracer/build/_build/Build.Steps.cs

```diff
@@ -269,6 +270,28 @@ partial class Build
         .DependsOn(CompileNativeSrcMacOs)
         .DependsOn(CompileNativeSrcLinux);
 
+    Target CppCheckNativeSrcUnix => _ => _
+        .Unlisted()
+        .OnlyWhenStatic(() => IsLinux || IsOsx)
+        .Executes(() =>
+        {
+            try
+            {
+                var (arch, ext) = GetUnixArchitectureAndExtension();
+                CppCheck.Value(arguments: $"cppcheck --inconclusive --std=c++11 --platform=native --project={NativeTracerProject.Path} --output-file={BuildDataDirectory}/{NativeTracerProject.Name}-cppcheck-{arch}.xml --xml --enable=warning,performance,portability");
+                CppCheck.Value(arguments: $"cppcheck --inconclusive --std=c++11 --platform=native --project={NativeTracerProject.Path} --output-file={BuildDataDirectory}/{NativeTracerProject.Name}-cppcheck-{arch}.txt --enable=warning,performance,portability");
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine("CppCheck cannot be run: {0}", ex.Message);
```

Comment:
I'm wondering why not fail the `CppCheckNativeSrcUnix` target if there was an exception?
(same for `CppCheckNativeLoader`)

Reply:
Hmm, because this was in the build step before having their own target and I copy/pasted the same thing, but you're right. I'll remove the try/catch